### PR TITLE
Write text + image content and HTML structure to portfolio home, gallery and blog pages

### PR DESCRIPTION
**Pull request by KathyFeiyang**
Write the primary text + image content, and corresponding HTML structures for porfolio
- Homepage contains personal information
- Gallery page contains photo albums
- Blog post page contains hidden talent posts
- Implement image responsiveness by selecting different resolutions based on vewport/device

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,109 @@
-<!DOCTYPE html>
-<html>
+<!doctype html>
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/index.html]: Keep this capitalized: go/htmlcssstyle#Document_Type

File: portfolio/src/main/webapp/blogs.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<!doctype html>
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/blogs.html]: This should be capitalized: go/htmlcssstyle#Document_Type

File: portfolio/src/main/webapp/blogs.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  <head>
+    <meta charset="UTF-8">
+	<meta name="Feiyang (Kathy) Yu's Blog Posts" content="Hidden talent blog posts of Feiyang (Kathy) Yu">
+	<link rel="stylesheet" type="text/css" href="/style.css">
+    <script src="/script.js"></script>
+	<title>Kathy's Blog Posts</title>
+  </head>
+  
+  <body>
+    <!--- Main section contains the blog post content -->
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/blogs.html]: contents

File: portfolio/src/main/webapp/blogs.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <main>
+      <h1>Kathy's Blog Posts</h1>
+      <nav role="navigation" aria-label="navigation bar">
+        <a href="#blog1">Blog 1</a><br>
+        <a href="#blog2">Blog 2</a><br>
+      </nav>
+	  <p>Welcome to my blog posts on hidden talents!</p>         
+            
+	  <h3 id="blog1">Hidden Talent NO.1</h3>
+	  <p>I know how to play badminton. I've always enjoyed the sport since I was a child, and I used to play it every week when I was in middle school. I haven't played it for a while now, so I'm quite rusty. But badminton is still my favorite sport regardless.</p>
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/blogs.html]: Break up long lines: go/htmlcssstyle#HTML_Line-Wrapping

File: portfolio/src/main/webapp/blogs.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+	  <h3 id="blog2">Hidden Talent NO.2</h3>
+	  <p>I've never had a nose bleed.</p>
+    </main>
+
+	<nav role="navigation" aria-label="navigation bar">
+	  <a href="/index.html">Go back to homepage</a>
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/blogs.html]: Keep the indentation consistent: go/htmlcssstyle#Indentation

File: portfolio/src/main/webapp/blogs.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+      <a href="/gallery.html">Check out gallery</a>
+      <a href="/index.html#contact_me">Contact Me</a>
+	</nav>
+
+    <footer>Last modified on <time>2020-05-26</time>.</footer>
+
+    </body>
+</html>
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/blogs.html]: Add a new line after this

File: portfolio/src/main/webapp/gallery.html [hunk 7]
```diff
@@ -0,0 +1,270 @@
+<!doctype html>
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/gallery.html]: Capitalize DOCTYPE

File: portfolio/src/main/webapp/gallery.html [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+      <h1>Kathy's Photo Gallery</h1>
+
+      <nav id="nav" role="navigation" aria-label="navigation bar">
+        <a href="#food">PHOTO ALBUM #1: Food</a><br>
+        <a href="#cuteness">PHOTO ALBUM #2: Cute Photos</a><br>
+        <a href="#places">PHOTO ALBUM #3: Beautiful Places</a><br>
+      </nav>
+
+      <p>Welcome to my photo gallery!</p>
+      <p>I enjoy documenting the little moments in life! A friend reunion, a delicious dish, a thoughtful text, a cool poster, ... I like photographing these things because they can vividly take me back in time.</p>
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/gallery.html]: Break up long lines

**Review by suparnabarua — CHANGES_REQUESTED** [review]
Adding initial set of comments. PTAL at the html style guide for reference.

File: portfolio/src/main/webapp/blogs.html (hunk 3 above)
Comment (KathyFeiyang) [reply portfolio/src/main/webapp/blogs.html]: Just to clarify, by "contents" do you mean that I should add the contents class attribute? Thank you!

Comment (suparnabarua) [reply portfolio/src/main/webapp/blogs.html]: Sorry, it was more of a language nitpick. I meant change "blog post content" to "blog post contents" :)

File: portfolio/src/main/webapp/blogs.html (hunk 6 above)
Comment (KathyFeiyang) [reply portfolio/src/main/webapp/blogs.html]: To confirm, are you referring to the practice that there should be a newline at the end of all source code files? I should add a newline after the closing html tag. Thanks!

File: portfolio/src/main/webapp/blogs.html [hunk 9]
```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8">
+	<meta name="Feiyang (Kathy) Yu's Blog Posts" content="Hidden talent blog posts of Feiyang (Kathy) Yu">
+	<link rel="stylesheet" type="text/css" href="/style.css">
```
Comment (kellyktan) [inline portfolio/src/main/webapp/blogs.html]: Not entirely sure, but it looks like you may be mixing tabs and spaces.  If so, please convert tabs to 2 spaces: https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: portfolio/src/main/webapp/blogs.html [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+  
+  <body>
+    <!--- Main section contains the blog post content -->
+    <main>
+      <h1>Kathy's Blog Posts</h1>
+      <nav role="navigation" aria-label="navigation bar">
+        <a href="#blog1">Blog 1</a><br>
+        <a href="#blog2">Blog 2</a><br>
+      </nav>
+	  <p>Welcome to my blog posts on hidden talents!</p>
```
Comment (kellyktan) [inline portfolio/src/main/webapp/blogs.html]: Get rid of any trailing whitespace: https://google.github.io/styleguide/htmlcssguide.html#Trailing_Whitespace

File: portfolio/src/main/webapp/index.html [hunk 11]
```diff
@@ -1,18 +1,109 @@
...
+<html lang="en-US">
   <head>
     <meta charset="UTF-8">
```
Comment (kellyktan) [inline portfolio/src/main/webapp/index.html]: I think this is supposed to be lower case? https://google.github.io/styleguide/htmlcssguide.html#Encoding https://google.github.io/styleguide/htmlcssguide.html#Capitalization

**Review by kellyktan — COMMENTED** [review]
Whoops, comments from before your latest commit, but I think still relevant?

File: portfolio/src/main/webapp/index.html (hunk 11 above)
Comment (KathyFeiyang) [reply portfolio/src/main/webapp/index.html]: I think DOCTYPE is a special case here: https://google.github.io/styleguide/htmlcssguide.html#Document_Type

File: portfolio/src/main/webapp/blogs.html (hunk 9 above)
Comment (KathyFeiyang) [reply portfolio/src/main/webapp/blogs.html]: Thanks! I'll look into it and make everything consistent

File: portfolio/src/main/webapp/index.html (hunk 11 above)
Comment (kellyktan) [reply portfolio/src/main/webapp/index.html]: *Oh I was referring to the "UTF-8", sorry for not being clearer!

Comment (KathyFeiyang) [reply portfolio/src/main/webapp/index.html]: Oh I see! Sorry about that. I'll make the change

**Review by kellyktan — APPROVED** [review]
LGTM!

File: portfolio/src/main/webapp/gallery.html [hunk 12]
```diff
@@ -0,0 +1,272 @@
...
+                     /images/cute_5 - Edited.jpg 200w"
+             sizes="(max-width: 600px) 200px,
+                    480px"/>
+        <figcaption>Real cat</figcaption>
+      </figure>
+      <div class="linking">
+        <a href="#nav"><h3>Go to top of the page</h3></a>
+      </div>
+
+	  <h3 id="places">PHOTO ALBUM #3: PLACES</h3>
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/gallery.html]: Fix the indentation here

**Review by suparnabarua — APPROVED** [review]
Just have one minor comment left

File: portfolio/src/main/webapp/gallery.html (hunk 12 above)
Comment (KathyFeiyang) [reply portfolio/src/main/webapp/gallery.html]: Will do!

**Pull request merged**